class_student.rb

```diff
@@ -0,0 +1,10 @@
+class Teacher < Person
```

Comment:
- [ ] This class name should be named `Student`. Please fix :wrench: it :+1:

Reply:
My bad